launch_frontend/launch_frontend/entity.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for Entity class."""
+
+import io
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Text
+from typing import Union
+
+from launch.utilities import is_a
+
+from pkg_resources import iter_entry_points
+
+frontend_entities = None
+
+
+class Entity:
+    """Single item in the intermediate front_end representation."""
+
+    @staticmethod
+    def load(
```

Comment:
@ivanpauno hmm why is an `Entity` parsing files instead of the `Parser`?

Reply:
It's loading a file, and not parsing it. It's true that this could be part of the parser, though.